src/Worker.Extensions.DurableTask/TaskOrchestrationContextExtensionMethods.cs

```diff
@@ -36,16 +39,13 @@ public static async Task<DurableHttpResponse> CallHttpAsync(this TaskOrchestrati
     /// <param name="content">Content passed in the HTTP request.</param>
     /// <param name="retryOptions">The retry option for the HTTP task.</param>
     /// <returns>A <see cref="Task{DurableHttpResponse}"/>Result of the HTTP call.</returns>
-    public static async Task<DurableHttpResponse?> CallHttpAsync(this TaskOrchestrationContext context, HttpMethod method, Uri uri, string? content = null, HttpRetryOptions? retryOptions = null)
+    public static Task<DurableHttpResponse> CallHttpAsync(this TaskOrchestrationContext context, HttpMethod method, Uri uri, string? content = null, HttpRetryOptions? retryOptions = null)
     {
-        DurableHttpRequest request = new DurableHttpRequest(
-            method: method,
-            uri: uri,
-            content: content,
-            httpRetryOptions: retryOptions);
+        DurableHttpRequest request = new DurableHttpRequest(method, uri);
```

Comment:
style nit: use object initializer

```suggestion
        DurableHttpRequest request = new DurableHttpRequest(method, uri)
        {
            Content = content,
            HttpRetryOptions = retryOptions,
        };
```